app/controllers/not_following_controller.rb

```diff
@@ -13,7 +14,10 @@ def not_following
 
   def set_users
     @user = params[:user]
-    @userlist = Functions::GetGithub.get_not_following_list(@user)
+    if !@user.empty?
+      @userlist = Functions::GetGithub.get_not_following_list(@user)
+    else
+      redirect_to root_path, notice: 'Kindly GitHub Username'
```

Comment:
Hey!
Thank you so much for helping me with this project :sunglasses:

Could you please just change the notice from 'Kindly Github Username' to something like 'Kindly enter your Github username' or 'Please enter your Github username'?

Right now it seems to be missing the action expected from the user!